plugins/out_s3/s3.c

```diff
@@ -841,6 +873,39 @@ static int cb_s3_init(struct flb_output_instance *ins,
         }
     }
 
+    tmp = flb_output_get_property("server-side-encryption-aws-kms-key-id", ins);
+    if (tmp) {
+        ctx->server_side_encryption_aws_kms_key_id = (char *) tmp;
+    }
+
+    tmp = flb_output_get_property("server-side-encryption", ins);
```

Comment:
After modifying to use camel_case on config_map, we need to fix using case_case here, too.